test/spark/README.md

```diff
@@ -16,34 +16,9 @@ sbt package
 The docker-compose yaml holds a service entry called `spark-submit` which uses the same settings as a Spark worker just with profile `command`, which will not start by default, unless "command" profile is specified.
 Using this entry and the volume mount to our workspace we submit the spark app.
 
-- `setup-exporter-test.sh` is responsible to setup the repository for the Exporter test. Repositories in the GC tests are created in the test itself.
-- The scripts `run-gc-test.sh` and `run-exporter-test.sh` are responsible to prepare the test environment and run the respective test `spark-submit` job.
-
 ### Exporter test
 
-The app itself is the word count example that store the information into csv format.  Loading the csv infomation and validate specific data in order to verify that the right data is loaded.
+- `setup-exporter-test.sh` is responsible to setup the repository for the Exporter test.
+- `run-exporter-test.sh` is responsible to prepare the test environment and run the respective test `spark-submit` job.
 
-### GC tests
```

Comment:
Are you removing this because it is understandable from the code?

Reply:
Yes, and also nothing related to the test is in this directory - so everything is under `esti/`. I considered removing this information elsewhere but I think it would be redundant.